### PR TITLE
update package.json

### PR DESCRIPTION
**Pull request by ikabir21**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation

enhancement

**What this PR does / why we need it**:
this PR updates the Dashboard version from 1.0.0 to the latest version 1.4.0
**Which issue(s) this PR fixes**:

Fixes #
[#627](https://github.com/sodafoundation/dashboard/issues/627)
**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by anvithks — APPROVED** [review]
LGTM
Thanks @ikabir21 .

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "soda-dashboard",
-    "version": "1.0.0",
+    "version": "1.4.0",
```
Comment (anvithks) [inline package.json:3]: @ikabir21 There was new pre-release out today. Can you please update the version to `1.4.1`.
thanks

**Review by anvithks — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ikabir21) [reply package.json:3]: sure

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**